governance/paper-process.md

```diff
@@ -173,3 +209,20 @@ strongly recommended the paper lead coordinate with TAG leadership to engage the
 CNCF team for posting a blog to summarize and link to the paper.  As community
 events occur, it is also recommended that the TAG coordinate a submission to
 community events (presentation) on the paper.
+
+#### Authorship, attribution and acknowledgements
```

Comment:
Thank you for addressing a topic that could get thorny in a succinct and fair way!